docs/user-guide/dev-environment.md

```diff
@@ -646,42 +667,42 @@ Loading a `cpe` module will do the following:
 - All HPE Cray PE modules will be updated so their default version is the
   one from the PE you have selected
 
-For example, if you have a code that uses the Gnu compiler environment, FFTW
-and NetCDF parallel libraries and you want to compile in the (non-default) 22.04
-programming environment, you would do the following:
+For example, if you have a code that uses the Gnu compiler environment, FFTW and
+NetCDF parallel libraries and you want to compile in the (non-default and older)
+22.12 programming environment, you would do the following:
 
-First, load the `cpe/23.09` module to switch all the defaults to the versions from
-the 22.04 PE. Then, swap to the GNU compiler environment and load the required library
+First, load the `cpe/22.12` module to switch all the defaults to the versions from
+the 22.12 PE. Then, swap to the GNU compiler environment and load the required library
 modules (FFTW, hdf5-parallel and NetCDF HDF5 parallel). The loaded module list shows they 
-are the versions from the 22.04 PE:
+are the versions from the 22.12 PE:
 
 
 ```bash
-module load cpe/23.09
+module load cpe/22.12
 ```
 
 Output:
 ```output
 
 The following have been reloaded with a version change:
-  1) PrgEnv-cray/8.3.3 => PrgEnv-cray/8.4.0             4) cray-mpich/8.1.23 => cray-mpich/8.1.27
-  2) cce/15.0.0 => cce/16.0.1                           5) craype/2.7.19 => craype/2.7.23
-  3) cray-libsci/22.12.1.1 => cray-libsci/23.09.1.1     6) perftools-base/22.12.0 => perftools-base/23.09.0
+  1) PrgEnv-cray/8.4.0 => PrgEnv-cray/8.3.3             4) cray-mpich/8.1.27 => cray-mpich/8.1.23
+  2) cce/16.0.1 => cce/15.0.0                           5) craype/2.7.23 => craype/2.7.19
+  3) cray-libsci/23.09.1.1 => cray-libsci/22.12.1.1     6) perftools-base/23.09.0 => perftools-base/22.12.0
 ```
 
 ```bash
 module load PrgEnv-gnu
 ```
 Output:
 ```output
-Lmod is automatically replacing "cce/16.0.1" with "gcc/11.2.0".
+Lmod is automatically replacing "cce/15.0.0" with "gcc/11.2.0".
 
 
-Lmod is automatically replacing "PrgEnv-cray/8.4.0" with "PrgEnv-gnu/8.4.0".
+Lmod is automatically replacing "PrgEnv-cray/8.3.3" with "PrgEnv-gnu/8.3.3".
 
 
 Due to MODULEPATH changes, the following have been reloaded:
-  1) cray-mpich/8.1.27
```

Comment:
a few of these areas where down grades are present as long as they are accurate that is fine.